app/controllers/outcome_groups_api_controller.rb

```diff
@@ -538,7 +538,7 @@ def unlink
 
   # @API List subgroups
   #
-  # List the immediate OutcomeGroup children of the outcome group. Paginated.
+  # A paginated list of the immediate OutcomeGroup children of the outcome group. Paginated.
```

Comment:
You should remove the mention of "Paginated" that already exists at the end of this line.